ADB/Airport Data/RouteAcquisition.py

```diff
@@ -0,0 +1,55 @@
+import os
+import requests
+import json
+from dotenv import load_dotenv
```

Comment:
I would recommend we use native Python functionality instead. [TOML support is relatively new](https://realpython.com/python-toml/). You can take some inspiration from: https://github.com/sustainableaviation/EcoPyLot/blob/8a6b7b5dff270293a38754459cb5f2812471245e/ecopylot/utils.py#L6